### ISSUE TITLE
iPython notebooks for API

### ISSUE THREAD

**Issue by andrewdhicks** [opendatacube/datacube-core#84]
Document functionality of the storage query and access module API.
### 1. intro
- Check connection
- Search database
- "What do I have?"
- get data
### 2. band math
- get data
- NDVI
- saving
- rgb, stretch bands & plot
### 3. seasonal stats
- time series stats
- group by seasons

**Comment by cet900** [comment]
Note we're happy to provide copies of or links to your demo ipynbs from the NCI training course for AGDC/VDI :)

**Comment by woodcockr** [comment]
We already have a set of these for AE/EE - why another set or is this the same set?

**Comment by woodcockr** [comment]
These notebooks are here:
https://github.com/data-cube/agdc-v2-examples/tree/master/notebooks/analytics_execution_engine

**Comment by omad** [comment]
I can't speak for everyone, but I hadn't seen those examples before since I've only been watching the main repo.

What's the reason for the separate repository? I think they'd be more discoverable and easier to maintain and use if they were kept in the main repo.

**Comment by cet900** [comment]
Have created a link from the training page to that repo. Thanks Rob!

**Comment by woodcockr** [comment]
Peter and I announced the examples repo when it was created a couple of weeks back. It was separated so others, who are EO AGDC users, but not core developers can create examples (different permissions on repos is needed).The main repo can have proper unit tests and integration tests (the examples that were there were out of date). If this is an issue, this can be changed, we just avoided cluttering the main repo in the first instance.

Regardless, the planning was that the AE/EE API be used for ipython notebooks and was on its work program. Why would it appear in a different work program? Would have been discussed at daily stand up and in slack as well from what I've heard. I raised an issue with the PM regarding potential duplicate work over two weeks ago on these.

Rather than duplicating work it would better to fix the tests that don't pass due to storage unit API and metadata changes which have been disabled in commit: n3b0dfacfae78eb13143e2bb7663d029fb74bcd04

@petewa and @jeremyh  were unable to reintegrate this as the breaking changes were discovered just prior to @petewa going on leave. The test was added to flag the problem so someone could fix the change - instead it was disabled which isn't very useful at the moment. The entire AE/EE and examples work against the milestone 2 storage API but not against what appears to be this weeks milestone 3 API. @peterwa is on leave so the milestone will not be met - which is reminiscent of milestone 2's outcome with late breaking changes on the API boundary.
